Yoast/javascript
# Deprecate obsolete morphology helper functions

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Deprecates obsolete morphology helper functions.

## Relevant technical choices:

* I changed the premium configuration dependency to the feature branch. So this should ideally be reverted back after the PR. Alternatively, it's also possible to create a temporary premium configuration branch of the same name as this PR (and set the dependency back to default).

## Test instructions

This PR can be tested by following these steps:

* Run `yarn test` and make sure that all tests still pass.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #
